src/CommandingOfficers/AW1/BH/Sturm.java

```diff
@@ -42,7 +39,7 @@ public instantiator()
             "Sturm (AW1)\n"
           + "A riddle within a shadow, revealing nothing.\n"
           + "???\n"
-          + "(1.3x/1.2x damage dealt/taken while COP isn't active, perfect movement outside snow.)"));
+          + "(1.3x/1.2x damage dealt/taken while COP isn't active, perfect movement outside cold weather.)"));
```

Comment:
outside "in" cold weather, or "except for" cold weather?

Sure, I know, and most people playing this will know, but the wording is less clear now.

I mean, it's probably fine. Taking AP English was a mistake.

Reply:
I guess I see what you're getting at with the ambiguity thing, but "snow" and "cold weather" don't seem meaningfully different to me in that regard.
One would ordinarily prefer to use a non-negative description (i.e. "perfect movement when warm"), but that's ambiguous in a much worse way in this case.

I guess I could say "in non-cold weather".

Reply:
I'd probably say "except in cold weather".